Gemfile

```diff
@@ -2,6 +2,7 @@
 source 'https://rubygems.org'
 
 gem 'colored'
+gem 'press_any_key'
```

Comment:
Is this gem required?  It may be easy enough to solve this problem with your own implementation, and while learning, perhaps that is a preferred situation.  This way you will not be surprised if this gem updates and makes breaking changes, and I suspect that perhaps the implementation is not going to be that large nor very complex.

This doesn't mean that you shouldn't find this gem and read the code to find out their opinions on solving this kind of problem.  That is a great exercise in itself.